docs/dev/table/connect.md

```diff
@@ -170,6 +181,50 @@ tableEnvironment
 {% endhighlight %}
 </div>
 
+<div data-lang="python" markdown="1">
+{% highlight python %}
+table_environment \
+    .connect(  # declare the external system to connect to
+             Kafka()
```

Comment:
The codestyle doesn't satisfy E126 continuation line over-indented for hanging indent and some similar problem in other code. Is there a better codestyle?